### PR TITLE
Download All submissions (based on #1181)

### PR DESCRIPTION
**Pull request by ajvb**
This PR takes #1181, rebases develop onto it, and adds further test.

This allows for a Journalist to click "Download all unread" from the submissions index view and download all unread submissions in one zip file.

Related to #1354

### CODE REVIEW

**Comment by fowlslegs** [comment]
Reviewing this now! If you're interested in starting on another one, https://github.com/freedomofpress/securedrop/issues/383 looks like an issue that might be suited to your skills and interests.

**Comment by fowlslegs** [comment]
To be clear, I'll take care of the small in-line issues too.

**Comment by ajvb** [comment]
@fowlslegs I'm more than happy to make style-related changes :)

File: securedrop/journalist.py [hunk 1]
```diff
@@ -627,20 +641,22 @@ def bulk_delete(sid, items_selected):
 
 def bulk_download(sid, items_selected):
     source = get_source(sid)
-    filenames = [store.path(sid, item.filename) for item in items_selected]
+    return bulk_dl(source.journalist_filename, items_selected)
+
```
Comment (ajvb) [inline securedrop/journalist.py]: fixed

**Comment by ajvb** [comment]
My bad, I missed your first comment before making the changes. Feel free to revert if it's easier.

**Comment by ajvb** [comment]
Closing this in favor of https://github.com/freedomofpress/securedrop/pull/1467

**Pull request closed without merging**